packages/hs-react-ui/src/components/Label/Label.stories.tsx

```diff
@@ -20,7 +20,13 @@ storiesOf('Label', module).add(
   () => (
     <Label
       labelText={text('labelText', 'This is the label text')}
-      color={select('Color', colors, colors.grayDark)}
+      color={select('Color', colors, colors.grayLight)}
+      colorValid={select('Color-valid', colors, colors.success)}
+      colorInvalid={select('Color-invalid', colors, colors.destructive)}
+      isRequired={boolean('isRequired', false)}
+      checkValidity={boolean('checkValidity', false)}
+      isValid={boolean('isValid', false)}
+      htmlFor="default"
     />
```

Comment:
now that we're accepting the children, open up component to have an hs-ui textInput inside. Something like this, but probably more detail:

```suggestion
    >
      <TextInput id={text('htmlFor', 'default')} />
    </Label>
```

Reply:
To get this working I had to add the id prop to textInput